modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -23,67 +23,74 @@ endif::[]
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
 = Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
 
-You can customize the install-config.yaml file to specify more details about
+You can customize the `install-config.yaml` file to specify more details about
 your {product-title} cluster's platform or modify the values of the required
 parameters.
 
+[NOTE]
+====
+If you want to deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter, you must manually configure the `install-config.yaml` for this environment. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
+====
+
 [source,yaml]
 ----
 apiVersion: v1
 baseDomain: example.com <1>
 compute: <2>
-- hyperthreading: Enabled <3>
-  name: worker
+- architecture: amd64
+  hyperthreading: Enabled <3>
+  name:  <worker_node>
+  platform: {}
   replicas: 3
-  platform:
-    vsphere: <4>
-      cpus: 2
-      coresPerSocket: 2
-      memoryMB: 8192
-      osDisk:
-        diskSizeGB: 120
 controlPlane: <2>
+- architecture: amd64
   hyperthreading: Enabled <3>
-  name: master
+  name: <parent_node>
+  platform: {}
   replicas: 3
-  platform:
-    vsphere: <4>
-      cpus: 4
-      coresPerSocket: 2
-      memoryMB: 16384
-      osDisk:
-        diskSizeGB: 120
 metadata:
-  name: cluster <5>
+  creationTimestamp: null
+  name: test <4>
 ifdef::network[]
 networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <9>
+  networkType: OVNKubernetes <8>
   serviceNetwork:
   - 172.30.0.0/16
 endif::network[]
 platform:
-  vsphere:
-    vcenter: your.vcenter.server
-    username: username
-    password: password
-    datacenter: datacenter
-    defaultDatastore: datastore
-    folder: folder
-    resourcePool: resource_pool <6>
-    diskType: thin <7>
-    network: VM_Network
-    cluster: vsphere_cluster_name <8>
+  vsphere: <5>
     apiVIPs: 
-      - api_vip
-    ingressVIPs: 
-      - ingress_vip
+      - 10.0.0.1
+      failureDomains:
```

Comment:
`failureDomains:` indentation wrong, 
`
    apiVIPs:
      - 10.0.0.1
      failureDomains:
      - <failure_domain_name>
      region: <default_region_name>
      server: <fully_qualified_domain_name>
      topology:
        computeCluster: "/<datacenter>/host/<cluster>"
        datacenter: <datacenter>
        datastore: "/<datacenter>/datastore/<datastore>"
         networks:
        - <VM_Network_name>
        resourcePool: "/<datacenter>/host/<cluster>/Resources/<resourcePool>" 
        folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>"
      zone: <default_zone_name>
    ingressVIPs:
    - 10.0.0.2
`
change to 
`
    apiVIPs:
      - 10.0.0.1
    failureDomains:
    - <failure_domain_name>
    region: <default_region_name>
    server: <fully_qualified_domain_name>
    topology:
      computeCluster: "/<datacenter>/host/<cluster>"
      datacenter: <datacenter>
      datastore: "/<datacenter>/datastore/<datastore>"
       networks:
      - <VM_Network_name>
      resourcePool: "/<datacenter>/host/<cluster>/Resources/<resourcePool>" 
      folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>"
    zone: <default_zone_name>
    ingressVIPs:
    - 10.0.0.2
`

Reply:
Hi @sgaoshang . I fixed the indentation.